Kitodo/src/main/webapp/newpages/inc_editProjects/detailsTab.xhtml

```diff
@@ -0,0 +1,226 @@
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+
+<ui:composition
+    xmlns="http://www.w3.org/1999/xhtml"
+    xmlns:h="http://xmlns.jcp.org/jsf/html"
+    xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+    xmlns:t="http://myfaces.apache.org/tomahawk"
+    xmlns:p="http://primefaces.org/ui"
+    >
+    <div id="tab1">
+        <h:panelGrid columns="2" rowClasses="top" id="projgrid2"
+                     styleClass="tableDetails ">
+
+            <!-- Titel -->
+            <h:outputLabel id="id17" for="titel"
+                           value="#{msgs.titel}"/>
+            <h:panelGroup id="id18">
+                <h:inputText id="titel"
+                             style="width: 200px;margin-right:15px"
+                             value="#{ProjekteForm.myProjekt.title}"
+                             required="true"/>
+                <t:message id="id19" for="titel" style="color: red"
+                           replaceIdWithLabel="true"/>
+            </h:panelGroup>
+
+            <!-- Mets als internes Speicherformat (ansonsten xStream)
```

Comment:
Why are you adding out commented lines to a new file? I think this should be removed or an explanation should be added.